pkg/apis/apps/v1alpha1/uniteddeployment_types.go

```diff
@@ -0,0 +1,229 @@
+/*
+Copyright 2019 The Kruise Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+// UnitedDeploymentSpec defines the desired state of UnitedDeployment
+type UnitedDeploymentSpec struct {
+	// Replicas is the totally desired number of replicas of all the owning workloads.
+	// If unspecified, defaults to 0.
```

Comment:
The mutating webhook handler below sets `replicas` default to 1

Reply:
done